src/main/index.ts

```diff
@@ -706,6 +706,9 @@ TrackAudioAfv.RegisterCallback((arg: string, arg2: string, arg3: string) => {
   }
 
   if (arg == AfvEventTypes.VoiceDisconnected) {
+    if (isInMiniMode()) {
+      toggleMiniMode();
+    }
```

Comment:
Noooooo please don't do this. TrackAudio shouldn't magically get gigantic and cover up other stuff on my screen because I closed CRC. It wasn't like this in prior releases.

Reply:
We will need to find a better way of displaying the `no active connection` message to the user then whilst in mini mode. We can't have the banner warped out of shape due to the size of the viewport. I'll try to think of a better way to show this, or maybe we silence these sorts of notifications in mini mode as long as they are not directly an error. Though as it stands toggling out of mini mode to display the 'error' (as this is how it is treated at the moment, which probably also needs changing) is the cleanest way to handle this sort of thing with the current way disconnects.

Also forgot to mention if its easier my discord is `georgebarlowr`, didn't see your previous message asking for it till now.

Reply:
I do know what you mean though, maybe this was a bit of an oversight on my behalf, let's try to handle this better.

Reply:
@neilenns I've just added a new commit it reverts this toggleMiniMode() and instead hides the error modal until you're back out of the mini mode 

Example below:

![gif](https://i.barlw.uk/s/934g0S.gif)